### PR TITLE
Clarify that `sky-row` now should be used within a `sky-fluid-grid` component.

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-fluid-grid-row-usage-clarification](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-fluid-grid-row-usage-clarification)

File: src/app/public/modules/fluid-grid/row.component.ts [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import { Component, Input } from '@angular/core';
 
+/**
+ * A previous implementation of the fluid grid allowed for the `sky-row` component to be used
```
Comment (blackbaud-johnly) [inline src/app/public/modules/fluid-grid/row.component.ts:4]:
```suggestion
 * Displays a row within the `sky-fluid-grid` wrapper. Previously, you could display a row
```

File: src/app/public/modules/fluid-grid/row.component.ts [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+ * without a wrapping `sky-fluid-grid` component; however, this is no longer officially supported.
```
Comment (blackbaud-johnly) [inline src/app/public/modules/fluid-grid/row.component.ts:5]:
```suggestion
 * without a wrapper, but we no longer officially support that option.
```

**Review by blackbaud-johnly — COMMENTED** [review]
I tweaked the wording based on our discussion, and I also pushed up an updated description for the wrapper.

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-fluid-grid-row-usage-clarification](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-fluid-grid-row-usage-clarification)

**Pull request merged**